http/src/request/application/set_command_permissions.rs

```diff
@@ -67,3 +89,114 @@ impl<'a> SetCommandPermissions<'a> {
 }
 
 poll_req!(SetCommandPermissions<'_>, ());
+
+#[cfg(test)]
+mod tests {
+    use super::SetCommandPermissions;
+    use crate::Client;
+    use twilight_model::{
+        application::command::permissions::{CommandPermissions, CommandPermissionsType},
+        id::{ApplicationId, CommandId, GuildId, RoleId},
+    };
+
+    #[test]
+    fn test_validation() {
+        let http = Client::new("token");
+
+        let permissions = vec![
+            (
+                CommandId(3),
+                CommandPermissions {
+                    id: CommandPermissionsType::Role(RoleId(4)),
+                    permission: true,
+                },
+            ),
+            (
+                CommandId(3),
+                CommandPermissions {
+                    id: CommandPermissionsType::Role(RoleId(4)),
+                    permission: true,
+                },
+            ),
+            (
+                CommandId(3),
+                CommandPermissions {
+                    id: CommandPermissionsType::Role(RoleId(4)),
+                    permission: true,
```

Comment:
There appears to be 12 of these. It could be
```rust
std::iter::repeat((
  CommandId(3),
  CommandPermissions {
    id: CommandPermissionsType::Role(RoleId(4)),
    permission: true,
  },
)).take(12).collect()
```

Reply:
ya i was trying to figure that out but i only found repeat ,, thank you